orderbook/test/orderbook-regression.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable max-params */
+/* eslint-disable max-len */
+
+'use strict';
+
+const assert = require('assert-diff');
+
+const RippleAPI = require('ripple-lib').RippleAPI;
+const OrderBook = require('../src/orderbook').OrderBook;
+
+const {normalizeCurrency} = require('../src/currencyutils');
+const regression = require('./fixtures/regression-data.json');
+const regressionResults = require('./fixtures/regression-data-results.json');
+const regressionTransactions = require('./fixtures/regression-data-transactions.json');
+
+
+describe.skip('regression test', function() {
```

Comment:
I don't know, maybe there is better word? Other tests are unit tests, but here I've recorded input and output of old `core` `OrderBook` class and checking new one against of what I've recorded.